samples/notebooks/python/10-multiple-results-per-prompt.ipynb

```diff
@@ -0,0 +1,317 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "68e1c158",
+   "metadata": {},
+   "source": [
+    "# Streaming Results"
```

Comment:
Can we put at the very top something along the lines of to help explain why this notebook exists?:

"In this notebook we show how you can in a single request, have the LLM model return multiple results per prompt. This is useful for running experiments where you want to evaluate the robustness of your prompt and the parameters of your config against a particular large language model."